it/src/test/java/org/apache/sling/scripting/bundle/tracker/it/ExampleBundleTeleportedIT.java

```diff
@@ -114,4 +115,18 @@ public void testAppsHello() throws LoginException {
             }
         }
     }
+
+    @Test
+    public void testRt() throws LoginException {
```

Comment:
Shouldn't this test be in a different Test Suite? Either a new one or https://github.com/apache/sling-org-apache-sling-scripting-bundle-tracker-it/blob/master/it/src/test/java/org/apache/sling/scripting/bundle/tracker/it/ExampleBundlePrecompiledIT.java?

Reply:
Moved